simue/simue.go

```diff
@@ -202,12 +203,22 @@ func ImsiStateMachine(profile *profctx.Profile, pCtx *profctx.ProfileUeContext,
 			pCtx.Log.Infoln("imsiStateMachine received result ")
```

Comment:
I think the changes should go here instead of in the individual `case` statements.
```suggestion
			pCtx.Log.Infoln("imsiStateMachine received result ")
			end_ts := time.Now()
			diff := end_ts.Sub(start_ts)
			pCtx.Log.Infof("[E2E], %v, %v, %v, %v, %v, %v, %v, %s\n", start_ts.Unix(), end_ts.Unix(), start_ts.UnixNano(), end_ts.UnixNano(), diff.Milliseconds(), end_ts.Format("15:04:05"), procedure, msg.GetEventType().String())
```
BTW, do you really need all the `times` in this log? Why not just printing the `diff` (without the other "times")?

Reply:
No actually not, It was for our other task. I will remove these.